R/util_data_summaries.R

```diff
@@ -77,17 +69,54 @@ ds.helper$default_col_funcs = list(
 ds.helper$default_categorical_col_funcs = list(
     "pct_NA" = ds.helper$pct_blank,
     "n_unique" = ds.helper$n_unique
-)   
+)  
 
+ds.helper$variable_type <- function(x){
+    # return variable type as
+    # (Note that NAs are ignored)
+    #  boolean - x contains exactly c(0,1) or c(TRUE,FALSE)
+    #  numeric - x is numbers 
+    #  categorical - x is a string or factor with up to 20 levels
```

Comment:
Could the 20 levels cutoff be a parameter that defaults to 20? Might not want to be wed to it.